packages/owl-symbolic/owl-symbolic.0.1.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "owlbarn"
+authors: [ "Liang Wang" "Jianxin Zhao" ]
+license: "Non-Profit OSL 3.0"
+homepage: "https://github.com/owlbarn/owl_symbolic"
+dev-repo: "git+https://github.com/owlbarn/owl_symbolic.git"
+bug-reports: "https://github.com/owlbarn/owl_symbolic/issues"
+doc: "https://owlbarn.github.io/owl_symbolic"
+description: "Connect Owl with other accelerators and numerical frameworks with symbolic maths."
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {>= "1.5.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11.0"}
```

The dune-project file uses lang 1.11

Reply:
Thanks for pointing out this issue. I'll close this PR and release an updated version.